### PR TITLE
ci: add jnb wsproxy + update dogfood template

### PR DESCRIPTION
**Pull request by johnstcn**

### CODE REVIEW

**Review by deansheather — COMMENTED** [review]
LGTM

**Review by matifali — APPROVED** [review]
LGTM. It's good that we now have a dogfood instance in South Africa.

Next we can add a wsproxy in Dubai.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -856,12 +856,14 @@ jobs:
...
           flyctl deploy --image "$IMAGE" --app sydney-coder --config ./.github/fly-wsproxies/sydney-coder.toml --env "CODER_PROXY_SESSION_TOKEN=$TOKEN_SYDNEY" --yes
           flyctl deploy --image "$IMAGE" --app sao-paulo-coder --config ./.github/fly-wsproxies/sao-paulo-coder.toml --env "CODER_PROXY_SESSION_TOKEN=$TOKEN_SAO_PAULO" --yes
+          flyctl deploy --image "$IMAGE" --app jnb-coder --config ./.github/fly-wsproxies/jnb-coder.toml --env "CODER_PROXY_SESSION_TOKEN=$TOKEN_JNB" --yes
         env:
           FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
           IMAGE: ${{ needs.build.outputs.IMAGE }}
           TOKEN_PARIS: ${{ secrets.FLY_PARIS_CODER_PROXY_SESSION_TOKEN }}
           TOKEN_SYDNEY: ${{ secrets.FLY_SYDNEY_CODER_PROXY_SESSION_TOKEN }}
           TOKEN_SAO_PAULO: ${{ secrets.FLY_SAO_PAULO_CODER_PROXY_SESSION_TOKEN }}
+          TOKEN_JNB: ${{ secrets.FLY_JNB_CODDER_PROXY_SESSION_TOKEN }}
```
Comment (deansheather) [inline .github/workflows/ci.yaml:866]:
```suggestion
          TOKEN_JNB: ${{ secrets.FLY_JNB_CODER_PROXY_SESSION_TOKEN }}
```

Comment (johnstcn) [reply .github/workflows/ci.yaml:866]: 🐟 CODDER

**Pull request merged**